conference.js

```diff
@@ -1093,6 +1094,9 @@ export default {
      * @param detailedFeedback detailed feedback from the user. Not yet used
      */
     sendFeedback(overallFeedback, detailedFeedback) {
+        APP.store.dispatch(sendJaasFeedbackMetadata(room, { score: overallFeedback,
```

Comment:
Since this one is returning promise, maybe you want to wait for it to finish as well before resolving/rejecting the result from the function?